core/dbt/parser/models.py

```diff
@@ -17,3 +20,48 @@ def resource_type(self) -> NodeType:
     @classmethod
     def get_compiled_path(cls, block: FileBlock):
         return block.path.relative_path
+
+    def render_update(
+        self, node: IntermediateNode, config: ContextConfig
+    ) -> None:
+
+        # normal dbt run
+        if not flags.USE_EXPERIMENTAL_PARSER:
```

Comment:
Something about how we're setting the flag value doesn't seem quite right. `flags.USE_EXPERIMENTAL_PARSER` has been returning `False` here, even when I run (in our internal analytics project):
```
$ dbt --use-experimental-parser parse --no-version-check
```

Reply:
Good catch! This was a "find and replace" error 🙃